src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -101,10 +101,19 @@ else if (currentPoint.pointType == EventPoint.POINT_TYPE_END) {
         if (finalList == null) {
             return (new ArrayList<TimeRange>());
         }
-
         finalList.sort(TimeRange.ORDER_BY_START);
 
-        return finalList;
+        ArrayList<TimeRange> removeDuplicates = new ArrayList<TimeRange>();
```

Comment:
nit: when we don't need special functions that only available in implements, prefer using interface, aka. List<TimeRange> removeDuplicates = new ArrayList<TimeRange>();